packages/components/src/avatar/src/Avatar.tsx

```diff
@@ -71,20 +71,14 @@ function AvatarImage({
 }
 
 const O365InitialsColorsForName = [
-    "#99B433",
-    "#6BA5E7",
-    "#E773BD",
-    "#00A300",
-    "#1E7145",
-    "#FF0097",
-    "#7E3878",
-    "#603CBA",
-    "#5E4B8B",
-    "#00ABA9",
-    "#2D89EF",
-    "#2B5797",
-    "#DA532C",
-    "#B91D47"
+    "#95B1FF",
```

Comment:
I missed this one yesterday are we ok with not supporting Microsoft initials colors?

Reply:
I think i would keep the old values until we completely rework the avatar. They are meant to match office 365

Reply:
If they've been changed to match the colors used in Hopper, we don't yet know whether the colors will carry over to O365.

<img width="263" alt="Capture d’écran, le 2023-10-11 à 13 10 11" src="https://github.com/gsoft-inc/wl-orbiter/assets/2379339/481eb575-16e9-4792-88e7-7baccc1b35ef">

Reply:
i guess we don't need the microsoft stuff anymore so the change is ok for now

Reply:
I think we will need to provide the possibility to pass custom colors for Hopper but I agree with Alex for Orbiter.